inc/packages/admin/namespace.php

```diff
@@ -235,11 +235,34 @@ function handle_direct_install() {
 	}
 
 	$skin = new WP_Upgrader_Skin();
-	$res = Packages\install_plugin( $id, $skin, $version );
-	var_dump( $res ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_dump
+	Packages\install_plugin( $id, $skin, $version );
 	exit;
 }
 
+/**
+ * Set slug to hashed slug from excaped slug-did.
```

Comment:
```suggestion
 * Set slug to hashed slug from escaped slug-did.
```
Stop dictating to your PC when writing docs :joy: